app/src/main/java/com/example/pinkorange/vibrato/SelectMusic.java

```diff
@@ -33,9 +33,8 @@ protected void onCreate(Bundle savedInstanceState) {
             public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
                 Intent intent = new Intent(view.getContext(), LiveWithSettings.class);
                 Song cur_song = songs.get(i);
-                // TODO: bundle the song item and send the whole thing to the next intent
-                // TODO: so that the next screen can use song to set title, artist, lyrics
                 intent.putExtra("song", cur_song);
```

Comment:
Comment for a future patch: maybe we can create a file with all of the strings we use to pass things around via intents, so it's all centralized and the strings are not hardcoded so we can change them. (not necessary in order to merge this PR though).